e2e/features/step_definitions/other-party-subscription.steps.ts

```diff
@@ -0,0 +1,98 @@
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_subscribeSms_' + action);
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyAppointeeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_tya', 'otherPartyAppointee123')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_email', 'otherpartyAppontee-test@mailinator.com')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyAppointeeSubscription_mobile', '01234567890')
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyRepresentativeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_tya', 'otherParty123')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_email', 'otherparty-test@mailinator.com')
+     await anyCcdPage.setValueByElementId('otherParties_0_otherPartyRepresentativeSubscription_mobile', '01234567890')
+ } else {
+    await anyCcdPage.clickElementById('subscriptions_appellantSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.clickElementById('subscriptions_appellantSubscription_subscribeEmail_' + action);
+
+    await anyCcdPage.clickElementById('subscriptions_representativeSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.clickElementById('subscriptions_representativeSubscription_subscribeEmail_' + action);
+
+    await anyCcdPage.clickElementById('subscriptions_appointeeSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.clickElementById('subscriptions_appointeeSubscription_subscribeEmail_' + action);
+
+    await anyCcdPage.clickElementById('subscriptions_jointPartySubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.clickElementById('subscriptions_jointPartySubscription_subscribeEmail_' + action);
+
+    await anyCcdPage.clickElementById('subscriptions_supporterSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.clickElementById('subscriptions_supporterSubscription_subscribeEmail_' + action);
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_subscribeEmail_' + action);
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyAppointeeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyAppointeeSubscription_subscribeEmail_' + action);
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyRepresentativeSubscription_wantSmsNotifications_' + action);
+     await anyCcdPage.clickElementById('otherParties_0_otherPartyRepresentativeSubscription_subscribeEmail_' + action);
+
+ }
```

Comment:
Incorrect formatting, leads me to believe this is the end of the function.

Reply:
Done